libzhlgen/test/functions/win32/1.6.9/freetype.zhl

```diff
@@ -15,6 +15,17 @@ static cleanup uint64_t freetype::easy_printCenter_DO_NOT_USE_DIRECTLY(int fontS
 ".8d4c240483e4f0ff71fc5589e55756535181ec????????8b5110":
 static cleanup __cdecl void freetype::easy_printAutoShrink(int fontId, float x, float y, int maxWidth, bool centered, const std::string& text);
 
+"55660fefc9660fefc089e55389cb83e4f083ec20":
+__thiscall void freetype::font_data::init(freetype::font_data *this, const void *buffer, int bufferSize, unsigned int h, bool glow);
+"5589e583e4f083ec108b4104890424e8????????c9c3":
```

Comment:
Linux (must be after `init` is declared)
`freetype::font_data::clean`
64: `!.8b7f04e9`
32: `!.8b4424048b400489442404`